src/utils/knowledgeartifacts.js

```diff
@@ -130,6 +131,44 @@ async function fetchValueSets(url, planDefinition, bundle) {
   }
 }
 
+/**
+ * Fetches any referenced Library resources from the planDefinition and saves to the database
+ *
+ * @param {string} url - fhir server base url to fetch from
+ * @param {PlanDefinition} planDefinition - PlanDefinition to search for referenced Library
+ * @param {Bundle} bundle - the Bundle the PlanDefinition was returned in,
+ *                          in case it already contains the Library we want
+ */
+async function fetchLibraries(url, plandefinition, bundle) {
+  const { library: libraryIds } = plandefinition;
+  if (!libraryIds || libraryIds.length === 0) return;
+
+  for (const id of libraryIds) {
```

Comment:
Are these always ids? I thought they were supposed to be canonical urls

Reply:
Hmm the library I used was exported from pathways-builder and just had the id in the library field. Should I change it to reference the url field?